src/components/Pages/Service.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { Fragment } from 'react';
```

Comment:
We think this is the reason the deploy is currently failing right?

Reply:
Yea, I'll upgrade react and react-dom